packages/dev/core/src/Materials/Textures/renderTargetTexture.ts

```diff
@@ -22,6 +21,7 @@ import "../../Engines/Extensions/engine.renderTarget";
 import "../../Engines/Extensions/engine.renderTargetCube";
 import { Engine } from "../../Engines/engine";
 import { _ObserveArray } from "core/Misc/arrayTools";
+import { DumpTools } from "core/Misc/dumpTools";

```

Comment:
Shouldn't this import (and the one above) be relative?

Reply:
I made a change to support package-based imports for different use-cases, but yes, it is always better to keep it relative anyhow.